openhands/server/routes/manage_conversations.py

```diff
@@ -122,7 +126,7 @@ async def _create_new_conversation(
             title=conversation_title,
             user_id=user_id,
             github_user_id=None,
-            selected_repository=selected_repository.full_name
+            selected_repository=selected_repository
             if selected_repository
             else selected_repository,
```

Comment:
```suggestion
            selected_repository=selected_repository,
```

it seems this conditional has become a no-op?

Reply:
Thank you!